group_vars/development

```diff
@@ -3,10 +3,10 @@ mysql_root_password: devpw
 web_user: vagrant
 
 wordpress_sites:
-  example.dev:
+  example.com:
     site_hosts:
       - example.dev
-    local_path: '../example.dev' # path targeting local project directory (relative to root/Vagrantfile)
+    local_path: '../example.com' # path targeting local project directory (relative to root/Vagrantfile)
```

Comment:
Thinking about it more, I realize I'm _still_ sometimes confused about the required name for my bedrock directory. Of course, the bedrock directory doesn't require any name in particular, so long as `local_path` matches.

It may minimize confusion to refer to `my-bedrock`. So, in `group_vars/<environment>`:
`local_path: '../`**my-bedrock**` # path targeting local`**BEDROCK**`directory'`

and in the README:
```
project/                   - Primary folder for the project
├── my-bedrock-ansible/    - This repo
└── my-bedrock/            - A Bedrock-based site (could name it 'example.com')
```

I realize that the bedrock directory is no longer just bedrock, it is a full-on "bedrock-based site", as you've called it, because of customizations, installation of dependencies, etc. So, technically you've been more accurate in calling it a 'bedrock-based site' or 'local project directory'. But I'm not sure that those phrases will always immediately translate in people's minds to 'my bedrock directory'. You're abstract and correct, but when I'm new, I need concrete.